src/features/MentorPage/MentorCard/MentorCardHeader.tsx

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components';
+import { ListCardProps } from '../../../../../mentor_card/src/features/MentorPage/ListCard/types';
```

Comment:
There are still few of these, output from grep:

 src/features/MentorPage/MentorCard/MentorCardHeaderBasicInfo.tsx:import { ListCardProps } from '../../../../../mentor_card/src/features/MentorPage/ListCard/types';
src/features/MentorPage/MentorCard/MentorContent.tsx:import { ListCardProps } from '../../../../../mentor_card/src/features/MentorPage/ListCard/types';
src/features/MentorPage/MentorCard/MentorCard.tsx:import { ListCardProps } from '../../../../../mentor_card/src/features/MentorPage/ListCard/types';
src/features/MentorPage/MentorCard/MentorCardHeader.tsx:import { ListCardProps } from '../../../../../mentor_card/src/features/MentorPage/ListCard/types';
src/features/MentorPage/MentorCard/MentorCardSkills.tsx:import { ListCardProps } from '../../../../../mentor_card/src/features/MentorPage/ListCard/types';

Reply:
aaah ok now I understand :D